rpcs3/Input/hid_pad_handler.cpp

```diff
@@ -104,6 +107,9 @@ hid_device* HidDevice::open()
 		hidDevice = hid_open_path(path.c_str());
 	}, false);
 #else
+	// Lock before calling "hid_open_path()"
+	std::lock_guard static_lock(g_hid_mutex);
```

Comment:
just use the unique lock above

Reply:
exactly this?

`std::unique_lock static_lock(g_hid_mutex, std::defer_lock); `

or without the deferring

Reply:
done